### DISCUSSION TITLE
How to set scale for a decimal editor dynamically

### DISCUSSION THREAD

**Discussion by rahooftkh** [serenity-is/Serenity#6240] [Help]
hi
 i want to set scale for a decimal editor dynamically from my dialogue 
,how to set this ,any idea??

**Comment by wezmag** [comment]
It uses [jQuery.autoNumeric](http://www.decorplanit.com/plugin/) plugin to control the number of decimal places.
You can change the number of decimal places as below.
```
(this.form.YourColumnName.element as any).autoNumeric('update', { mDec: 5 })
```
mDec: number of decimal places.
